src/mono/mono/mini/aot-compiler.c

```diff
@@ -15623,9 +15623,9 @@ mono_aot_assemblies (MonoAssembly **assemblies, int nassemblies, guint32 jit_opt
 		dedup_methods = g_hash_table_new (NULL, NULL);
 	}
 
-	if (aot_opts.compiled_methods_outfile) {
-		if (g_ensure_directory_exists (aot_opts.compiled_methods_outfile) == FALSE) {
-			fprintf (stderr, "AOT : failed to create the directory to save the compiled method names: %s\n", aot_opts.compiled_methods_outfile);
+	if (aot_opts.trimming_eligible_methods_outfile) {
+		if (g_ensure_directory_exists (aot_opts.trimming_eligible_methods_outfile) == FALSE) {
+			fprintf (stderr, "AOT : failed to create the directory to save the compiled method names: %s\n", aot_opts.trimming_eligible_methods_outfile);
```

Comment:
maybe update this message so it's "trimmable method names" now

Reply:
Good catch! Updated.